File: LABS/LR1-2/Пак 5 Задача 4.pas
program qq;
var
coordinate:string;
digit, letter:char;
digit_num, letter_num, sum:integer;

begin
  
  writeln('Введите координаты точки: ');
  read(coordinate);
  
  letter := coordinate[1];
  digit := coordinate[2]; 
  
  letter_num := ord(letter);
  digit_num := ord(digit);
  
  sum := letter_num + digit_num;
  
  if sum mod 2 = 0 then
    writeln('BLACK')
  else
    writeln('WHITE');
end.
  

File: LABS/LR1-2/Пак 5 Задача 4.py
coordinate = input('Введите координаты точки: ')

letter = coordinate[0]
digit = coordinate[1]

letter_num = ord(letter)
digit_num = ord(digit)

sum = letter_num + digit_num

if sum % 2 == 0:
    print('BLACK')
else:
    print('WHITE')